frontend/app/controllers/Joiner.scala

```diff
@@ -144,26 +144,39 @@ object Joiner extends Controller with ActivityTracking
 
       val flowSelected = CheckoutFlowVariant.getFlowVariantFromRequestCookie(request).getOrElse(CheckoutFlowVariant.A)
 
-      Ok(flowSelected match {
-        case CheckoutFlowVariant.A => views.html.joiner.form.payment(
-          plans,
-          countryCurrencyWhitelist,
-          identityUser,
-          pageInfo,
-          trackingPromoCode = validTrackingPromoCode,
-          promoCodeToDisplay = validDisplayablePromoCode,
-          Some(countryGroup))
-        case CheckoutFlowVariant.B => views.html.joiner.form.paymentB(
+
+      Ok(
+      if(paypalTest){
```

Comment:
This big if statement is ugly, but it is just to support testing and will disappear when we release this stuff